src/locales/en.json

```diff
@@ -112,7 +112,9 @@
       }
     },
     "geolocationTracking": {
-      "androidBackgroundPermissionMessage": "Recording your movements requires to ≪ {backgroundPermissionOptionLabel} ≫"
+      "androidBackgroundPermissionMessage": "Recording your movements requires to ≪ {backgroundPermissionOptionLabel} ≫",
+      "androidNotificationTitle": "Location service on",
+      "androidNotificationDescription": "Your journey will be recorded into your Cozy"
```

Comment:
No ES version?

Also this mention "Cozy", so I suppose we should get an alternative text in extra-language files